Makefile

```diff
@@ -1,6 +1,6 @@
 TESTIMONY_TOKENS="caseautomation, casecomponent, caseimportance, caselevel, caseposneg, description, expectedresults, id, requirement, setup, subtype1, subtype2, steps, teardown, testtype, upstream, title"
 TESTIMONY_MINIMUM_TOKENS="id, requirement, caseautomation, caselevel, casecomponent, testtype, caseimportance, upstream"
-PYTEST_OPTIONS=--verbose
+PYTEST_OPTIONS="--verbose --driver=Remote"
```

Comment:
Where is this being used? We are not using pytest-selenium anymore so I think this is not needed.

Reply:
this is used whenever when run `test-qpc*` or any variants in the Makefile. It can be overridden on the command line but is a handy default.